public/styles/styles.css

```diff
@@ -0,0 +1,101 @@
+/* styles.css */
+
+body {
+  background-color: Coral; /*sanity check*/
```

Comment:
do you still need this sanity check? or is it no longer a sanity check, and is now your actual style?